sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -161,6 +173,117 @@ public EventDirectorySteps(
           webDriverHelpers.clickOnWebElementBySelector(FIRST_CHECKBOX_EVENT_DIRECTORY);
           webDriverHelpers.waitForPageLoaded();
         });
+
+    When(
+        "I search last created Event by {string} option filter in Event Group Directory",
+        (String searchCriteria) -> {
+          String searchText = "";
+          switch (searchCriteria) {
+            case "EVENT_ID":
+              searchText = apiState.getCreatedEvent().getUuid();
+              break;
+            case "TITLE":
+              searchText = apiState.getCreatedEvent().getEventTitle();
+              break;
+            case "LOCATION":
+              searchText = apiState.getCreatedEvent().getTypeOfPlace();
+              break;
+            case "DESCRIPTION":
+              searchText = apiState.getCreatedEvent().getEventDesc();
+              break;
+          }
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.fillInWebElement(EVENT_GROUP_FREE_TEXT_EVENT_INPUT, searchText);
+        });
+
+    When(
+        "I chose Region option in Event Group Directory",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
+          String region = apiState.getCreatedEvent().getEventLocation().getRegion().getUuid();
+          webDriverHelpers.selectFromCombobox(
+              EVENT_REGION_COMBOBOX_INPUT, RegionsValues.getValueFor(region));
```

Comment:
is this method correct? shouldn't be get value for uuid? (as above for similar enum)